arai60/subarray-sum-equal-k/step2.py

```diff
@@ -0,0 +1,28 @@
+"""
+https://leetcode.com/problems/subarray-sum-equals-k/solutions/1759711/python-simple-python-solution-using-prefixsum-and-dictionary/?envType=problem-list-v2&envId=me1nua2e
+-> 命名が参考になった
+https://leetcode.com/problems/subarray-sum-equals-k/solutions/5143436/560-subarray-sum-equals-k-brute-force-tle-hashmap-prefix-sum-approaches-better-comments/?envType=problem-list-v2&envId=me1nua2e
+-> running_numはなるほど。
+https://github.com/hayashi-ay/leetcode/pull/31/files
+-> ほぼ同じ
+https://github.com/ryoooooory/LeetCode/pull/3/files
+-> java。javaのmapってgetOrDefaultとかあるんだ。便利~~
```

Comment:
https://docs.python.org/3/library/stdtypes.html#dict.get
> get(key, default=None)¶
> Return the value for key if key is in the dictionary, else default. If default is not given, it defaults to None, so that this method never raises a [KeyError](https://docs.python.org/3/library/exceptions.html#KeyError).

Reply:
Pythonでもdefault指定できたんですね～
dictのドキュメントも読んでまとめようと思います